OracleDatabase/11.2.0.2/Vagrantfile

```diff
@@ -20,37 +20,66 @@ VAGRANTFILE_API_VERSION = "2"
 BOX_URL = "https://oracle.github.io/vagrant-boxes/boxes"
 BOX_NAME = "oraclelinux/7"
 
-# define hostname
-NAME = "oracle11g-xe-vagrant"
-
 unless Vagrant.has_plugin?("vagrant-proxyconf")
   puts 'Installing vagrant-proxyconf Plugin...'
   system('vagrant plugin install vagrant-proxyconf')
 end
 
-# get host time zone for setting VM time zone, if possible
-# can override in env section below
-offset_sec = Time.now.gmt_offset
-if (offset_sec % (60 * 60)) == 0
-  offset_hr = ((offset_sec / 60) / 60)
-  timezone_suffix = offset_hr >= 0 ? "-#{offset_hr.to_s}" : "+#{(-offset_hr).to_s}"
-  SYSTEM_TIMEZONE = 'Etc/GMT' + timezone_suffix
-else
-  # if host time zone isn't an integer hour offset, fall back to UTC
-  SYSTEM_TIMEZONE = 'UTC'
+# Define constants
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+  # Use vagrant-env plugin if available
+  if Vagrant.has_plugin?("vagrant-env")
+    config.env.load(".env.local", ".env") # enable the plugin
+  end
+
+  # VM name
+  NAME = default_s('NAME', 'oracle11g-xe-vagrant')
+
+  # Memory for the VM (in MB, 2048 MB = 2 GB)
+  MEMORY = default_i('MEMORY', 2048)
```

Comment:
Seems like we are duplicating the default values, once here and once in the `.env` file. Is that necessary, as this could easily be overlooked by someone and lead to unexpected behavior?

Reply:
Having the default values in the `Vagrantfile` allows it to work as is, without the user installing the `vagrant-env` plugin or setting any environment variables. If no customization is needed, then people can just run `git clone` / `vagrant up`.